test/pylib/util.py

```diff
@@ -240,3 +240,10 @@ async def finish():
         return 0
 
     return finish
+
+async def wait_for_view(cql: Session, name: str, node_count: int, timeout: int = 120):
+    async def view_is_built():
+        done = await cql.run_async(f"SELECT COUNT(*) FROM system_distributed.view_build_status WHERE status = 'SUCCESS' AND view_name = '{name}' ALLOW FILTERING")
+        return done[0][0] == node_count or None
+    deadline = time.time() + timeout
+    await wait_for(view_is_built, deadline)
```

Comment:
Note that wait_for() defaults to check period of 1 seconds. It makes me sad to think that we think that a 1 second sleep is negligible in sleeps. It makes me happy once again that this test/pylib is *not* used in test/cql-pytest and even more so in test/alternator.